src/platform-implementation-js/dom-driver/gmail/views/gmail-thread-view.ts

```diff
@@ -211,6 +211,10 @@ class GmailThreadView {
     '2023_11_16': '* > .nH',
   };
 
+  _subjectAISuggestionsContainerSelectors = {
```

Comment:
```suggestion
  #subjectAISuggestionsContainerSelectors = {
```

Reply:
Yeah, I see your point however I followed the pattern for consistency so I'd rather refactor this file separately to keep the PR footprint minimal.